frontend/src/pages/groupSettings/GroupSettings.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { useEffect, useState } from 'react';
+import {
+  ActionGroup,
+  Button,
+  Form,
+  FormGroup,
+  PageSection,
+  PageSectionVariants,
+  Text,
+  HelperText,
+  HelperTextItem,
+  Alert,
+  AlertActionCloseButton,
+} from '@patternfly/react-core';
+import ApplicationsPage from '../ApplicationsPage';
+import { useDispatch } from 'react-redux';
+import { addNotification, detectUser } from '../../redux/actions/actions';
+import './GroupSettings.scss';
+import { fetchGroupsSettings, updateGroupsSettings } from 'services/groupSettingsService';
+import { GroupsConfig, GroupsConfigField, MenuItemStatus } from 'types';
+import { MenuOptionMultiSelect } from 'components/MultiSelection';
+
+const TITLE = `User and group settings`;
```

Comment:
We don't tend to do this else where just add the string.  We do have to add i18n support in the future we could probabaly remove these then.